fastapi-crud/app/events.py

```diff
@@ -25,8 +27,8 @@ class Host(BaseModel):
 
 class ResponseData(BaseModel):
     name: str
-    address: str
-    schedule: datetime.datetime
+    address: Optional[str]
+    schedule: Optional[datetime.datetime]
     host: Optional[Host]
```

Comment:
I think the idea here is to use e.g. `dbqueries.CreateEventResult` directly without having to wrap the result again.

Reply:
Thanks for the note on this. I will circle back and use the generated model.

Reply:
Thanks and sorry for the late notice! As you're using single-file mode, I think your generated model names are sufferring from [this issue](https://github.com/edgedb/edgedb-python/issues/397) too, it's probably better to use something like `import ... CreateEventResult as ...EventResult` (or `EventResult = db_queries.CreateEventResult`) to rename the models for now.

Reply:
Just pushed some changes addressing this, @fantix. Check it out when you have a chance and let me know if I need to revisit.